docs/ceh.md

```diff
@@ -0,0 +1,21 @@
+# nf-core/configs: Center for Evolutionary Hologenomics & Section for Evolutionary Genomics (hologenomics partition on HPC) Configuration
+
+The profile is configured to run with Singularity version 3.6.3-1.el7 which is part of the OS installtion and does not need to be loaded as a module. 
+
+Before running the pipeline you will need to load Java, Miniconda and Nextflow. You can do this by including the commands below in your SLURM/sbatch script:
+
+```bash
+## Load Java, Miniconda and Nextflow environment modules
+module purge
+module load lib
+module load java/v1.8.0_202-jdk miniconda nextflow/v20.07.1.5412
```

Comment:
```suggestion
module load java/v1.8.0_202-jdk nextflow/v20.07.1.5412
```

Reply:
Why are you removing the miniconda module? It needs to be listed so it can be loaded ;)